src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -18,7 +18,9 @@
     @Override
     public CommandResult executePrimitive(Model model, CommandHistory history) {
         requireNonNull(model);
-        model.resetData(new TaskManager());
+        TaskManager clearedTaskManager = new TaskManager();
```

Comment:
Would prefer if you implement this as on a lower level of abstraction instead i.e. in the model class as a model#clearTasksOnly.

You can ignore the review and merge otherwise. LGTM